packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -148,8 +155,15 @@ class CollapsibleMenuView extends React.Component {
     customProps.className);
     const menuButtonClassName = cx(
       'menu-button',
-      { hidden: this.menuHidden },
+      { hidden: !this.collapsedMenuAlwaysShown && this.menuHidden },
     );
+    let visibleChildren = children;
+    let hiddenChildren = alwaysCollapsedMenuItems;
+
+    if (this.hiddenStartIndex >= 0) {
+      visibleChildren = React.Children.toArray(children);
+      hiddenChildren = visibleChildren.splice(this.hiddenStartIndex).concat(prepopulatedBaseDivider).concat(hiddenChildren);
```

Comment:
It looks like we're always adding the `prepopulatedBaseDivider` even if there are not any `alwaysCollapsedMenuItems`. We should only add the divider if always collapsed menu items exist.

Reply:
Matt, you are absolutely and 100% right. I feel ashamed I missed this scenario. Pushed the fix. It seems the wdio tests couldn't get this. It might be under the treshold.

Reply:
Don't feel ashamed, we all miss stuff. 😄